meetings/2025-03-05.md

```diff
@@ -0,0 +1,121 @@
+* HZ: No
+* HZ: It is not that when Inrupt donated it to OWF and wanted it to be the story. It wanted cross pollination and the API to be inproved. We don't care about the wallet but about the storage. It is open source code licensed under Apache.
+* eP: Yesterday I joined the credentials CG, where someone from Octa and Sam from Google were presenting the digital credentials API, and discussed the differences with FedCM. There are multiple protocols, like OpenID4VP. It uses FIDO etc. and QR codes. What is the overlap between that and the wallets you are discussing now? OM th
+* MdJ: I think it is pretty related. AFAIK Credentials CG is about interfacing wallets with the web. You could have a boarding pass or driver's license on your phone, and CG describes how it could be shared in the browser to the website. You share it either with the QR code or with NFC to some device, similar to payments. They also add how to present it via JS API. What I'm doing is about wallets, but not necessarly adding interaces and web page but between them and storage servers.
+* HZ: The wallet API, the backend is documented and open. Did you take a look at it and would it be usefull in your project? It could be useful to have multiple implementations. There is also ane from South American company which is doing attachments on VCs.
+* eP: In the case you're describing, connecting to storage, how is the user involved?
+* MdJ: The user would not see much of a difference between sharing a credential on their device vs sharing one in their storage. You could have them in both location. It could be transparent to the user. The Verifiers would not receive the whole thing in the QR code or NFC interaction. They could receive a read capability for example. 
+* eP: so where the wallet is makes the main difference, device vs pod
+* MdJ: ... There might be a big overlap with organizational wallets. What I will build will be a combination of a mobile app with pod.
+* eP: It would be interesting to see how it plays in local-first setup. Where VCs would be replicaterd.
+* HZ: Do you know that the open wallet Inrupt donated to OWF is white labeled?
+* MdJ: I'm using API but not building the app.
+* HZ: We want to make it pluggable on the backend. On the frontend you may want to change what is being displayed etc.
+
+### PATCH
+* MdJ: The way I see it, https://github.com/solid/specification/issues/715 sounds like it would not be a minor bugfix, so I think it would be more a task for LWS WG, not for Solid CG.
+* MdJ: Regarding https://github.com/solid/specification/issues/715 I would initially say it sounds like a minor bugfix, but after reading about how difficult this would be for implementors, given the apparent unavailability of off-the-shelf parsers that could deal with this, I don't think it's within remit for Solid CG to merge a PR like this, so again, this feels to me like something we should leave for LWS WG to work on.
```

Comment:
```suggestion
* MdJ: The way I see it, https://github.com/solid/specification/issues/715 sounds like it would not be a minor bug fix, so I think it would be more a task for LWS WG, not for Solid CG.
* MdJ: Regarding https://github.com/solid/specification/issues/715, I would initially say it sounds like a minor bug fix, but after reading about how difficult this would be for implementers, given the apparent unavailability of off-the-shelf parsers that could deal with this, I don't think it's within remit for Solid CG to merge a PR like this, so again, this feels to me like something we should leave for LWS WG to work on.
```